bftengine/include/bftengine/ReconfigurationCmd.hpp

```diff
@@ -0,0 +1,98 @@
+// Concord
+//
+// Copyright (c) 2021 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+#include "ReservedPagesClient.hpp"
+#include "Serializable.h"
+#include "concord.cmf.hpp"
+#include <optional>
+
+namespace bftEngine {
+class ReconfigurationCmd : public bftEngine::ResPagesClient<ReconfigurationCmd, 1> {
```

Comment:
What if we have several reconfiguration requests that are relevant to this epoch, wouldn't it be better to save a list of the actual reconfiguration requests (and our current epoch number)

Reply:
fixed